File: output/ChatGPT/codenet/Python/Java/s683277023.java
import java.util.Arrays;
import java.util.Scanner;

public class s683277023 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int k = scanner.nextInt();
        int[] li = new int[n];
        for (int i = 0; i < n; i++) {
            li[i] = scanner.nextInt();
        }
        Arrays.sort(li);
        int[] tmp = Arrays.copyOfRange(li, 0, k);
        int sum = 0;
        for (int num : tmp) {
            sum += num;
        }
        System.out.println(sum);
    }
}

File: output/ChatGPT/codenet/Python/C++/s683277023.cpp
#include <iostream>
#include <vector>
#include <algorithm>

int main() {
    int n, k;
    std::cin >> n >> k;
    std::vector<int> li(n);
    for (int i = 0; i < n; i++) {
        std::cin >> li[i];
    }
    std::sort(li.begin(), li.end());
    std::vector<int> tmp(li.begin(), li.begin() + k);
    int sum = 0;
    for (int i = 0; i < k; i++) {
        sum += tmp[i];
    }
    std::cout << sum << std::endl;
    return 0;
}